contracts/v0.1/deployments/collected-addresses.json

```diff
@@ -0,0 +1,152 @@
+{
```

Comment:
@nick-bisonai How about splitting the collected addresses into two files? One file would be for `Data Feed` and the other one for the rest. `Data Feed` would have a general format that is easy to read and integrate to other services (e.g. orakl-frontend in https://github.com/Bisonai/orakl-frontend/blob/1a805b64cf97ca6b054649ba1542c48f34504523/src/utils/constants.ts#L223-L629) if necessary.

For the `Data Feed` json format, I think the following would be nice

```json
{
    "ADA-USDT": {
        "baobab": {
            "aggregator": "0x70cDE6bE67486017C52215Ad5d6740ce8EaBC9b8",
            "proxy": "0xB9C1839A40cED59Fb9e55Eb52a3B8B7E62aF1568"
        },
        "cypress": {
            "aggregator": "0x235587EA94b2fe15FfFf0577303E5F0Cf13C29Ab",
            "proxy": "0x04a77b347d1e0FD6FA9af328aB0232F3F2Be05C0",
        },
    }
}
```